npm-wrapper/README.md

```diff
@@ -0,0 +1,32 @@
+## accessibility-insights-for-android-service-bin
+
+This NPM package is a thin wrapper around the [Accessibility Insights for Android Service](../README.md) APK file. The package bundles a copy of the APK and exports the path of the bundled APK (and its associated NOTICE file) as the only library exports.
+
+### Usage
+
+```js
+import { noticePath, apkPath } from 'accessibility-insights-for-android-service-bin';
+
+console.log(`Absolute path of the APK bundled with the package: ${apkPath}`);
+console.log(`Absolute path of the NOTICE file for the APK: ${noticePath}`);
+```
+
+This wrapper package is intended for consumption by [Accessibility Insights for Android](https://github.com/microsoft/accessibility-insights-web); we make no guarantees about its API stability or fitness for other purposes.
+
+### Versioning
+
+This wrapper package **does not use semantic versioning**. Its versioning strategy is modeled after the same one used by `chromedriver` and `7zip-bin`; the major and minor version will always match the major and minor version of the APK, and the patch version will increment with each released version of the NPM wrapper package for a given major/minor version of the APK.
```

Comment:
I'm wondering if we should expose the APK version as a property on the package. This would allow a consumer to get the APK version without using adb. It seems like the major and minor version should be enough for most purposes though, so it feels OK to defer until we really need it

Reply:
I just updated the readme as you were writing this comment to reflect that we actually decided to keep the wrapper and APK version in exact sync, which I think removes the need for this